app/styleguide/page.tsx

```diff
@@ -50,18 +50,18 @@ import {
   CardMutedText,
 } from "@/components/Card/card";
 
-import {BrandColor, Example, handleDelete,} from "./sg-components";
+import { BrandColor, Example, handleDelete, } from "./sg-components";
 
 import TypeAheadInput from '@/components/TypeAheadInput';
-import TypeAheadWithOther from '@/components/TypeAheadWithOther';
-import {AffiliationsDocument} from '@/generated/graphql';
+import TypeAheadWithOther from '@/components/Form/TypeAheadWithOther';
+import { AffiliationsDocument } from '@/generated/graphql';
 
 import "./styleguide.scss";
 import SectionHeaderEdit from "@/components/SectionHeaderEdit";
 import QuestionEdit from "@/components/QuestionEdit";
 import SubHeader from "@/components/SubHeader";
 import TooltipWithDialog from "@/components/TooltipWithDialog";
-import {ModalOverlayComponent} from '@/components/ModalOverlayComponent';
+import { ModalOverlayComponent } from '@/components/ModalOverlayComponent';
```

Comment:
Apologies for the added spaces from my `prettier` formatting. But I think the added space is ok, but let me know what you think.